### PR TITLE
click-to-play overlay advanced sample for amp-video

### PR DESCRIPTION
**Pull request by aghassemi**
Closes : https://github.com/ampproject/amphtml/issues/8947
Addresses: https://github.com/ampproject/amphtml/issues/8896

eowxsps93uh

### CODE REVIEW

File: src/30_Advanced/Click-to-play_overlay_for_amp-video.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      opacity: 1;
+    }
+  </style>
+</head>
+<body>
+
+  <!-- ## Markup -->
+  <!--
+    The overlay is an absolutely positioned `div` that sits on top of the
+    `amp-video` element and fills the parent Player container. It
```
Comment (sebastianbenz) [inline src/30_Advanced/Click-to-play_overlay_for_amp-video.html]: s/Player/player

File: src/30_Advanced/Click-to-play_overlay_for_amp-video.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        <h3 class="video-title">Title of the video</h3>
+
+        <div class="play-icon"
+             role="button" tabindex="0"
+             on="tap:myOverlay.hide, myVideo.play"></div>
+
+        <amp-img class="poster-image"
+                 layout="fill"
+                 src="https://unsplash.it/1080/720?image=666"></amp-img>
+      </div>
```
Comment (sebastianbenz) [inline src/30_Advanced/Click-to-play_overlay_for_amp-video.html]: nit: for consistency add a line break after the div.

File: src/30_Advanced/Click-to-play_overlay_for_amp-video.html (hunk 1 above)
Comment (aghassemi) [inline src/30_Advanced/Click-to-play_overlay_for_amp-video.html]: done

File: src/30_Advanced/Click-to-play_overlay_for_amp-video.html (hunk 2 above)
Comment (aghassemi) [inline src/30_Advanced/Click-to-play_overlay_for_amp-video.html]: done

**Comment by sebastianbenz** [comment]
great sample! - thanks!

**Pull request merged**